### PR TITLE
Profile source name in request

### PR DESCRIPTION
**Pull request by bobcaprice**

### CODE REVIEW

**Comment by bobcaprice** [comment]
[Changing name private doesn't update common:source-name in a timely manor](https://trello.com/c/WfEWj0EG/3449-changing-name-private-doesn-t-update-common-source-name-in-a-timely-manor)

File: orcid-core/src/main/java/org/orcid/core/manager/impl/SourceNameCacheManagerImpl.java [hunk 1]
```diff
@@ -39,58 +41,61 @@
...
-                            sourceNameCache.put(new Element(cacheKey, StringUtils.EMPTY));
-                        } else {
-                            sourceNameCache.put(new Element(cacheKey, sourceName));
-                        }                        
-                    }                    
+                            sourceName = getClientSourceName(sourceId);
+                            if (sourceName != null) {
+                                sourceNameCache.put(new Element(cacheKey, sourceName));
+                            } else {
+                                sourceName = getProfileSourceNameFromDb(sourceId);
```
Comment (rcpeters) [inline orcid-core/src/main/java/org/orcid/core/manager/impl/SourceNameCacheManagerImpl.java]: getProfileSourceNameFromRequest?

File: orcid-web/src/main/webapp/static/javascript/angularOrcid.js [hunk 2]
```diff
@@ -53,13 +53,13 @@
 	__webpack_require__(1);
```
Comment (amontenegro) [inline orcid-web/src/main/webapp/static/javascript/angularOrcid.js]: Hey @bobcaprice, could you please rollback these changes?

@rcpeters FYI

Comment (amontenegro) [inline orcid-web/src/main/webapp/static/javascript/angularOrcid.js]: Nevermind

**Pull request merged**